src/main/java/frc/robot/commands/ArmCommand.java

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import com.revrobotics.AbsoluteEncoder;
+import com.revrobotics.SparkMaxAbsoluteEncoder;
+import com.revrobotics.CANSparkMax.ControlType;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.config.Config;
+import frc.robot.subsystems.ArmSubsystem;
+import frc.robot.subsystems.SwerveSubsystem;
+import java.util.Arrays;
+
+public class ArmCommand extends CommandBase {
+  String level;
+  boolean isTop;
+  double integralMinimum = -0.5;
+  double integralMaximum = 0.5;
+  double z;
+  double nodeX;
+  double armSpeed = 0.2;
+  double defaultAngle1 = Math.PI; // angle in radians of joint 1 in default position (vertical)
+  double defaultAngle2 = Math.PI/36; // angle in radians of joint 2 in default position (angle of 5 degrees)
+  double[] defaultAngles;
+  double targetAngularDistanceBottomArm;
+  double targetAngularDistanceTopArm;
+
+  /** Creates a new ArmExtend. */
+  
+  public ArmCommand(String level, boolean isTop) {
+    this.level = level;
+    this.isTop = isTop;
```

Comment:
what is the purpose of isTop? Can this be a boolean that is derived from the level being given? (i.e. if the level provided is "the top level" then you set the variable that way).

Reply:
isTop is a boolean that helps us test each angles since we only have one motor
needed so that the motor controllers don't get confused
We will remove it later